opencga-catalog/src/main/java/org/opencb/opencga/catalog/monitor/executors/LocalExecutor.java

```diff
@@ -41,10 +43,10 @@ public LocalExecutor() {
     public void execute(Job job, String token) throws Exception {
         Runnable runnable = () -> {
             try {
-                ExecutorConfig executorConfig = getExecutorConfig(job);
+                ExecutorConfig executorConfig = ExecutorConfig.getExecutorConfig(job);
 
                 logger.info("Ready to run {}", job.getCommandLine());
-                Command com = new Command(getCommandLine(job, token));
+                Command com = new Command(job.getCommandLine() + " --session-id " + token);
```

Comment:
I'd like to keep the method "getCommandLine" in the BatchExecutor interface, either as default or as a static method. Or, at least, create a constant for the "--session-id".
This may change in the future.